perses/app/relative_setup.py

```diff
@@ -263,9 +262,9 @@ def __init__(self, ligand_input, old_ligand_index, new_ligand_index, forcefield_
 
         # Create SystemGenerator
         from openmmforcefields.generators import SystemGenerator
-        forcefield_kwargs = {'removeCMMotion': False, 'ewaldErrorTolerance': self._pme_tol, 'nonbondedMethod': self._nonbonded_method,'constraints' : app.HBonds, 'hydrogenMass' : self._hmass}
```

Comment:
i missed something; why is te nonbonded method not being specified anymore?

Reply:
Good spot, this is to do with openmm-forcefields, where whatever nonbonded method we chose was being overwritten by the default for `periodic_forcefield_kwargs` (which is PME anyway, but the reason that RF wasn't working before)

openmm-forcefield fix: https://github.com/openmm/openmmforcefields/pull/109
perses issue: https://github.com/choderalab/perses/issues/661

Reply:
so if i wanted `NoCutoff` for vacuum, what would i do?